uniffi/src/bindings/kotlin/templates/ObjectTemplate.kt

```diff
@@ -1,25 +1,34 @@
 
-class {{ obj.name()|class_name_kt }}(handle: Long) {
-    private var handle: AtomicLong = AtomicLong(handle)
-    {%- for cons in obj.constructors() %}
+class {{ obj.name()|class_name_kt }}(
+    internal val handle: Long
```

Comment:
`handle` is never going to change, so removed `AtomicLong`.

Reply:
This makes sense to me, but I do wonder why we tend to use AtomicLong for handles in Application Services 🤔

Reply:
Now changed back to `AtomicLong`, so we can zero out the handle when we're destroying it.